pkg/commands/set_cluster.go

```diff
@@ -93,7 +93,8 @@ func SetClusterFields(fields []string, cluster *api.Cluster, instanceGroups []*a
 			cluster.Spec.NodePortAccess = append(cluster.Spec.NodePortAccess, kv[1])
 		case "spec.kubernetesVersion":
 			cluster.Spec.KubernetesVersion = kv[1]
-
+		case "spec.masterPublicName":
+			cluster.Spec.masterPublicName = kv[1]
```

Comment:
I think this should be `cluster.Spec.MasterPublicName` (capital M)

Reply:
fixed typo in 78d16f246758253f1b9a898bd1f5023c0d3484cf - `/s/(Spec.)m(asterPublicName)/$1M$2/`

pull-kops-e2e-kubernetes-aws — Job failed. [Details](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kops/7160/pull-kops-e2e-kubernetes-aws/1141685394924900352/)

```
I0620 12:37:27.976] /root/.cache/bazel/_bazel_prow/ae5d1f01453377487c630b230ced7d61/sandbox/linux-sandbox/836/execroot/__main__/pkg/commands/set_cluster.go:97:16: cluster.Spec.masterPublicName undefined (type "k8s.io/kops/pkg/apis/kops".ClusterSpec has no field or method masterPublicName, but does have MasterPublicName)
```